# Estonian translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-10-18 03:11+0000\n"
"PO-Revision-Date: 2011-10-14 09:13+0000\n"
"Last-Translator: tabbernuk <boamaod@gmail.com>\n"
"Language-Team: Estonian <et@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2011-10-18 14:52+0000\n"
"X-Generator: Launchpad (build 14157)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:416 ../UpdateManager/Core/utils.py:416
msgid "0 kB"
msgstr "0 kB"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:419 ../UpdateManager/Core/utils.py:419
msgid "1 kB"
msgstr "1 kB"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:422 ../UpdateManager/Core/utils.py:422
#, python-format
msgid "%.0f kB"
msgstr "%.0f kB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:425 ../UpdateManager/Core/utils.py:425
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:210 ../DistUpgrade/distro.py:438
#, python-format
msgid "Server for %s"
msgstr "Server %s jaoks"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:228 ../DistUpgrade/distro.py:234
#: ../DistUpgrade/distro.py:250
msgid "Main server"
msgstr "Põhiserver"

#: ../DistUpgrade/distro.py:254
msgid "Custom servers"
msgstr "Kohandatud serverid"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:141
msgid "Could not calculate sources.list entry"
msgstr "Sources.list kirjet pole võimalik tõlgendada"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:244
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Ühtegi pakifaili ei leitud. Võib-olla ei ole see Ubuntu plaat või on valele "
"protsessoriarhitektuurile?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:287
msgid "Failed to add the CD"
msgstr "CD lisamine nurjus"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:288
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"CD lisamisel tekkis viga, uuendamine katkeb. Palun teata sellest kui veast, "
"kui tegemist on õige Ubuntu CDga.\n"
"\n"
"Veateade oli:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:138
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Eemalda halvas seisus tarkvarapakett"
msgstr[1] "Eemalda halvas seisus tarkvarapaketid"

#: ../DistUpgrade/DistUpgradeCache.py:141
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Paketi '%s' paigaldus sisaldab vasturääkivusi ning see tuleks uuesti "
"paigaldada, kuid selle arhiivi ei leitud. Kas eemaldada see pakett praegu ja "
"jätkata?"
msgstr[1] ""
"Pakettide '%s' paigaldus sisaldab vasturääkivusi ning need tuleks uuesti "
"paigaldada, kuid nende arhiive ei leitud. Kas eemaldada need paketid praegu "
"ja jätkata?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:239
msgid "The server may be overloaded"
msgstr "Server võib olla ülekoormatud"

#: ../DistUpgrade/DistUpgradeCache.py:352
msgid "Broken packages"
msgstr "Katkised paketid"

#: ../DistUpgrade/DistUpgradeCache.py:353
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Sinu süsteem sisaldab katkiseid pakette, mida pole võimalik antud tarkvaraga "
"parandada. Palun paranda need synaptic'u või apt-get'i abil enne jätkamist."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:648
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Uuendamise rehkendamisel esines lahendamatu viga:\n"
"%s\n"
"\n"
" Selle võis põhjustada:\n"
" * uuendamine Ubuntu eelväljalaske versioonile\n"
" * Ubuntu eelväljalaske versiooni kasutamine\n"
" * mitteametlike pakkide kasutamine, mida ei jaga Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:658
msgid "This is most likely a transient problem, please try again later."
msgstr "See on üsna tõenäoliselt mööduv probleem, proovi hiljem uuesti."

#: ../DistUpgrade/DistUpgradeCache.py:661
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:666
#: ../UpdateManager/UpdateManager.py:1023
msgid "Could not calculate the upgrade"
msgstr "Uuenduse arvutamine polnud võimalik"

#: ../DistUpgrade/DistUpgradeCache.py:717
msgid "Error authenticating some packages"
msgstr "Viga mõnede pakettide usaldusväärsuse kinnitamisel"

#: ../DistUpgrade/DistUpgradeCache.py:718
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Mõnede pakettide usaldusväärsuse kinnitamine polnud võimalik. See võib olla "
"mööduv võrguprobleem, nii et võid hiljem uuesti proovida. All järgneb "
"kontrollimata pakettide nimekiri."

#: ../DistUpgrade/DistUpgradeCache.py:738
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Eemaldamiseks on märgistatud pakett '%s', mis on aga eemaldamise mustas "
"nimekirjas."

#: ../DistUpgrade/DistUpgradeCache.py:742
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Oluline pakett '%s' on märgistatud eemaldamiseks."

#: ../DistUpgrade/DistUpgradeCache.py:751
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Halvas nimekirjas oleva versiooni '%s' paigaldamise katse"

#: ../DistUpgrade/DistUpgradeCache.py:869
#, python-format
msgid "Can't install '%s'"
msgstr "Paketi '%s' paigaldamine pole võimalik"

#: ../DistUpgrade/DistUpgradeCache.py:870
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:881
msgid "Can't guess meta-package"
msgstr "Meta-paketi arvamine pole võimalik"

#: ../DistUpgrade/DistUpgradeCache.py:882
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Sinu süsteemis pole paketti 'ubuntu-desktop', 'kubuntu-desktop', 'xubuntu-"
"desktop' või 'edubuntu-desktop' ja polnud võimalik tuvastada, millist Ubuntu "
"versiooni sa kasutad.\n"
"Enne jätkamist paigalda synaptic'u või apt-get'i abil mõni ülalnimetatud "
"pakettidest."

#: ../DistUpgrade/DistUpgradeController.py:100
msgid "Reading cache"
msgstr "Vahemälu lugemine"

#: ../DistUpgrade/DistUpgradeController.py:205
msgid "Unable to get exclusive lock"
msgstr "Lukustamine ebaõnnestus"

#: ../DistUpgrade/DistUpgradeController.py:206
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Tavaliselt tähendab see, et teine paketihaldur (nt apt-get või aptitude) "
"töötab. Palun sulge esmalt see teine rakendus."

#: ../DistUpgrade/DistUpgradeController.py:239
msgid "Upgrading over remote connection not supported"
msgstr "Kaugühenduse kaudu uuendamine ei ole toetatud."

#: ../DistUpgrade/DistUpgradeController.py:240
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Te teostate uuendust kaug-ssh-ühenduse kaudu kasutajaliidesega, mis ei toeta "
"seda. Palun proovige tekstirežiimis uuendamist käsuga 'do-release-upgrade'.\n"
"\n"
"Uuendamine katkestatakse. Palun proovige uuesti ilma ssh ühenduseta."

#: ../DistUpgrade/DistUpgradeController.py:254
msgid "Continue running under SSH?"
msgstr "Kas jätkata SSH all?"

#: ../DistUpgrade/DistUpgradeController.py:255
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"See seanss paistab töötavat üle ssh. Ssh üle ei ole soovitav uuendusi "
"teostada, sest ebaõnnestumise korral on taastamine keerulisem.\n"
"\n"
"Kui jätkate, siis käivitatakse veel üks ssh daemon pordis '%s'.\n"
"Kas soovite jätkata?"

#: ../DistUpgrade/DistUpgradeController.py:269
msgid "Starting additional sshd"
msgstr "Täiendava sshd käivitamine"

#: ../DistUpgrade/DistUpgradeController.py:270
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Et teha nurjumise korral taastamine lihtsamaks, käivitatakse lisaks veel üks "
"sshd pordil '%s'. Kui miski peaks viltu minema esimese ssh-ga, saad ikka "
"ühenduda teise külge.\n"

#: ../DistUpgrade/DistUpgradeController.py:278
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Kui kasutad tulemüüri, pead selle pordi ajutiselt lahti tegema. Kuna see "
"võib olla ohtlik, et tehta seda automaatselt. Pordi saad avada näiteks "
"käsuga '%s'"

#: ../DistUpgrade/DistUpgradeController.py:349
#: ../DistUpgrade/DistUpgradeController.py:394
msgid "Can not upgrade"
msgstr "Uuendamine pole võimalik"

#: ../DistUpgrade/DistUpgradeController.py:350
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "See tööriist ei toeta uuendamist '%s'-lt '%s'-le."

#: ../DistUpgrade/DistUpgradeController.py:359
msgid "Sandbox setup failed"
msgstr "Liivakasti ülesseadmine nurjus"

#: ../DistUpgrade/DistUpgradeController.py:360
msgid "It was not possible to create the sandbox environment."
msgstr "Liivakastikeskkonna loomine ei olnud võimalik."

#: ../DistUpgrade/DistUpgradeController.py:366
msgid "Sandbox mode"
msgstr "Liivakastirežiim"

#: ../DistUpgrade/DistUpgradeController.py:367
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"See uuendus toimub liivakastirežiimis (testi-). Kõik muudatused kirjutatakse "
"'%s'-le ning kaovad järgmisel taaskäivitusel.\n"
"\n"
"*Mitte ükski* muudatus, mis alates praegusest hetkest kuni järgmise "
"taaskäivituseni kirjutatakse süsteemikausta, pole püsiv."

#: ../DistUpgrade/DistUpgradeController.py:395
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Sinu python'i paigaldus on vigane. Palun paranda nimeviide '/usr/bin/python'."

#: ../DistUpgrade/DistUpgradeController.py:421
msgid "Package 'debsig-verify' is installed"
msgstr "Tarkvarapakett 'debsig-verify' on paigaldatud"

#: ../DistUpgrade/DistUpgradeController.py:422
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Uuendamine ei saa jätkuda, kui see pakett on paigaldatud.\n"
"Palun eemalda see kõigepealt synaptikuga või käsuga 'apt-get remove debsig-"
"verify' ning käivita uuendus uuesti."

#: ../DistUpgrade/DistUpgradeController.py:432
msgid "Include latest updates from the Internet?"
msgstr "Kas lisada värskeimad uuendused internetist?"

#: ../DistUpgrade/DistUpgradeController.py:433
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Uuenduste süsteem võib internetist automaatselt alla laadida uuendusi ning "
"uuendamise käigus need paigaldada. Kui sul on internetiühendus, on see on "
"tungivalt soovitatud.\n"
"\n"
"Selliseks uuendamiseks kulub kauem aega, aga kui see lõpeb, on su süsteem "
"täiesti ajakohane. Sa võid seda ka mitte teha, kuid siis peaksid esimesel "
"võimalusel värskeimad uuendused ikkagi paigaldama.\n"
"Kui vastad siin eitavalt, ei kasutata internetti üldse."

#: ../DistUpgrade/DistUpgradeController.py:653
#, python-format
msgid "disabled on upgrade to %s"
msgstr "keelatud %s-le uuendamisel"

#: ../DistUpgrade/DistUpgradeController.py:680
msgid "No valid mirror found"
msgstr "Ühtegi sobivat peeglit ei leitud"

#: ../DistUpgrade/DistUpgradeController.py:681
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Tarkvaraallika skaneerimisel ei leitud uuenduse peeglit. See võib juhtuda, "
"kui kasutad sisepeegelit või kui peeglis olev teave on aegunud.\n"
"\n"
"Kas tahad oma 'sources.list' faili ikka üle kirjutada? Kui valid siin 'Jah', "
"muudetakse kõik '%s' kirjed '%s'-ks.\n"
"Kui valid 'Ei', katkestatakse uuendus."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:701
msgid "Generate default sources?"
msgstr "Kas genereerida vaikimisi allikad?"

#: ../DistUpgrade/DistUpgradeController.py:702
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Pärast 'sources.list' skaneerimist ei leitud sobivat kirjet '%s' jaoks.\n"
"\n"
"Kas vaikimisi kirjed '%s' jaoks tuleks lisada? Kui valid 'Ei', katkestatakse "
"uuendamine."

#: ../DistUpgrade/DistUpgradeController.py:737
msgid "Repository information invalid"
msgstr "Varamu informatsioon vigane"

#: ../DistUpgrade/DistUpgradeController.py:738
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug using the command 'ubuntu-bug update-manager' in a "
"terminal."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:746
msgid "Third party sources disabled"
msgstr "Kolmandate osapoolte allikad keelatud"

#: ../DistUpgrade/DistUpgradeController.py:747
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Mõned kolmandate osapoolte kirjed 'sources.list'-is keelati. Pärast uuendust "
"võid need jälle lubada, kasutades tööriista 'software-properties' või oma "
"paketihaldurit."

#: ../DistUpgrade/DistUpgradeController.py:787
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Vasturääkivustega pakett"
msgstr[1] "Vasturääkivustega paketid"

#: ../DistUpgrade/DistUpgradeController.py:790
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Paketi '%s' olek sisaldab vasturääkivusi ning see tuleb uuesti paigaldada, "
"kuid arhiivi ei leitud. Palun paigalda see käsitsi uuesti või eemalda "
"süsteemist."
msgstr[1] ""
"Pakettide '%s' olekud sisaldavad vasturääkivusi ning need tuleb uuesti "
"paigaldada, kuid arhiive ei leitud. Palun paigalda need käsitsi uuesti või "
"eemalda süsteemist."

#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Error during update"
msgstr "Uuendamise ajal ilmnes viga."

#: ../DistUpgrade/DistUpgradeController.py:839
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Uuendamisel esines viga. Tavaliselt on selleks võrguprobleem, kontrolli "
"võrguühendust ja proovi uuesti."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:848
#: ../UpdateManager/UpdateManager.py:747
msgid "Not enough free disk space"
msgstr "Kõvakettal pole piisavalt vaba ruumi"

#: ../DistUpgrade/DistUpgradeController.py:849
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Uuendamine katkes. Uuendamiseks on vaja kokku %s vaba ruumi kettal '%s'. "
"Vabasta veel vähemalt %s ruumi kettal '%s'. Tühjenda prügikast ja eemalda "
"ajutised paketid eelmistest paigaldustest käsuga 'sudo apt-get clean'."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:878
#: ../DistUpgrade/DistUpgradeController.py:1630
msgid "Calculating the changes"
msgstr "Muudatuste rehkendamine"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:910
msgid "Do you want to start the upgrade?"
msgstr "Kas alustada uuendamist?"

#: ../DistUpgrade/DistUpgradeController.py:976
msgid "Upgrade canceled"
msgstr "Uuendamine katkestatud"

#: ../DistUpgrade/DistUpgradeController.py:977
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Uuendamine katkestatakse nüüd ning esialgne süsteemi olukord taastatakse. Sa "
"võid uuendada mõni teine kord."

#: ../DistUpgrade/DistUpgradeController.py:983
#: ../DistUpgrade/DistUpgradeController.py:1109
msgid "Could not download the upgrades"
msgstr "Uuenduste allalaadimine polnud võimalik"

#: ../DistUpgrade/DistUpgradeController.py:984
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Uuendamine katkestati. Palun, veendu oma netiühenduse või paigaldusmeedia "
"töökorras ja proovi uuesti! Kõik siiani alla laaditud failid säilitati."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1060
#: ../DistUpgrade/DistUpgradeController.py:1097
#: ../DistUpgrade/DistUpgradeController.py:1202
msgid "Error during commit"
msgstr "Toimingu ajal esines viga"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1062
#: ../DistUpgrade/DistUpgradeController.py:1099
#: ../DistUpgrade/DistUpgradeController.py:1241
msgid "Restoring original system state"
msgstr "Süsteemi algse oleku taastamine"

#: ../DistUpgrade/DistUpgradeController.py:1063
#: ../DistUpgrade/DistUpgradeController.py:1078
#: ../DistUpgrade/DistUpgradeController.py:1100
msgid "Could not install the upgrades"
msgstr "Uuenduste paigaldamine polnud võimalik"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1068
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Uuendamine katkes. Sinu arvuti võib olla praegu kasutamatu. Praegu käivitati "
"taastamisoperatsioon (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1073
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug using the command 'ubuntu-bug update-manager' in a "
"terminal and include the files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1110
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Uuendamine katkes. Kontrolli internetiühendust või paigaldusmeediumi ning "
"proovi uuesti. "

#: ../DistUpgrade/DistUpgradeController.py:1190
msgid "Remove obsolete packages?"
msgstr "Kas eemaldada iganenud paketid?"

#: ../DistUpgrade/DistUpgradeController.py:1191
#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Keep"
msgstr "_Hoia alles"

#: ../DistUpgrade/DistUpgradeController.py:1191
msgid "_Remove"
msgstr "_Eemalda"

#: ../DistUpgrade/DistUpgradeController.py:1203
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr "Puhastamise käigus esines viga. Täpsem info alumises teates. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1279
msgid "Required depends is not installed"
msgstr "Vajalik sõltuvus pole paigaldatud"

#: ../DistUpgrade/DistUpgradeController.py:1280
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Vajalik sõltuvus '%s' pole paigaldatud. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1527
#: ../DistUpgrade/DistUpgradeController.py:1597
msgid "Checking package manager"
msgstr "Paketihalduri kontrollimine"

#: ../DistUpgrade/DistUpgradeController.py:1532
msgid "Preparing the upgrade failed"
msgstr "Uuendamise ettevalmistamine ebaõnnestus"

#: ../DistUpgrade/DistUpgradeController.py:1533
msgid ""
"Preparing the system for the upgrade failed. Please report this using the "
"command 'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1549
msgid "Getting upgrade prerequisites failed"
msgstr "Uuendamise eelduste hankimine ebaõnnestus"

#: ../DistUpgrade/DistUpgradeController.py:1550
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug using the command 'ubuntu-bug update-manager' in "
"a terminal and include the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "Updating repository information"
msgstr "Varamuinfo uuendamine"

#: ../DistUpgrade/DistUpgradeController.py:1588
msgid "Failed to add the cdrom"
msgstr "Cdromi lisamine nurjus"

#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Sorry, adding the cdrom was not successful."
msgstr "Kahjuks cdromi lisamine nurjus."

#: ../DistUpgrade/DistUpgradeController.py:1617
msgid "Invalid package information"
msgstr "Vigane paketiinfo"

#: ../DistUpgrade/DistUpgradeController.py:1618
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1636
#: ../DistUpgrade/DistUpgradeController.py:1689
msgid "Fetching"
msgstr "Allalaadimine"

#: ../DistUpgrade/DistUpgradeController.py:1642
#: ../DistUpgrade/DistUpgradeController.py:1693
msgid "Upgrading"
msgstr "Uuendamine"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1647
#: ../DistUpgrade/DistUpgradeController.py:1695
#: ../DistUpgrade/DistUpgradeController.py:1702
#: ../DistUpgrade/DistUpgradeController.py:1713
msgid "Upgrade complete"
msgstr "Uuendamine valmis"

#: ../DistUpgrade/DistUpgradeController.py:1648
#: ../DistUpgrade/DistUpgradeController.py:1696
#: ../DistUpgrade/DistUpgradeController.py:1703
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "Uuendus on lõpetatud, kuid uuendamise ajal tekkisid vead."

#: ../DistUpgrade/DistUpgradeController.py:1655
msgid "Searching for obsolete software"
msgstr "Aegunud tarkvara otsimine"

#: ../DistUpgrade/DistUpgradeController.py:1664
msgid "System upgrade is complete."
msgstr "Süsteemi uuendamine on valmis."

#: ../DistUpgrade/DistUpgradeController.py:1714
msgid "The partial upgrade was completed."
msgstr "Osaline uuendamine lõpetati."

#: ../DistUpgrade/DistUpgradeQuirks.py:201
msgid "evms in use"
msgstr "evms on kasutusel"

#: ../DistUpgrade/DistUpgradeQuirks.py:202
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Sinu süsteem kasutab 'evms' köitehaldurit kohas /proc/mounts. 'Evms' "
"tarkvara pole enam toetatud, lülita see välja ja käivita uuendamine uuesti."

#: ../DistUpgrade/DistUpgradeQuirks.py:506
msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:508
msgid ""
"The support in Ubuntu 11.04 for your intel graphics hardware is limited and "
"you may encounter problems after the upgrade. Do you want to continue with "
"the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:528
#: ../DistUpgrade/DistUpgradeQuirks.py:556
#: ../DistUpgrade/DistUpgradeQuirks.py:583
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Uuendamine võib eemaldada töölauaefektid ja vähendada mängude ning teiste "
"graafikamahukate rakenduste jõudlust."

#: ../DistUpgrade/DistUpgradeQuirks.py:532
#: ../DistUpgrade/DistUpgradeQuirks.py:560
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"See arvuti kasutab praegu NVIDIA graafikadraiverit nimega 'nvidia'.  Kahjuks "
"ei ole sellest draiverist versiooni, mis töötaks Ubuntu 10.04 LTS-is.\n"
"\n"
"Kas tahad sellest hoolimata jätkata?"

#: ../DistUpgrade/DistUpgradeQuirks.py:587
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"See arvuti kasutab praegu AMD graafikadraiverit nimega 'fglrx'.  Kahjuks ei "
"ole sellest draiverist versiooni, mis töötaks Ubuntu 10.04 LTS-is.\n"
"\n"
"Kas tahad sellest hoolimata jätkata?"

#: ../DistUpgrade/DistUpgradeQuirks.py:617
msgid "No i686 CPU"
msgstr "Pole i686 CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:618
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Sinu süsteem kasutab i586 arhitektuuriga protsessorit (CPU) või CPU ei oma "
"\"cmov\" laiendust. Kõik paketid on koostatud optimeerituna minimaalselt "
"i686 arhitektuurile. Sellise riistvaraga ei ole võimalik süsteemi paigaldada "
"uut Ubuntu väljalaset."

#: ../DistUpgrade/DistUpgradeQuirks.py:654
msgid "No ARMv6 CPU"
msgstr "Ei leitud ARMv6 protsessorit"

#: ../DistUpgrade/DistUpgradeQuirks.py:655
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Su süsteem kasutab ARM protsessorit, mis on vanem kui ARMv6 arhitektuur. "
"Kõik paketid karmic'us koostati optimeeritult vähemalt ARMv6 arhitektuurile. "
"Selle raudvaraga ei ole võimalik Ubuntut uuendada."

#: ../DistUpgrade/DistUpgradeQuirks.py:675
msgid "No init available"
msgstr "Init pole saadaval"

#: ../DistUpgrade/DistUpgradeQuirks.py:676
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Tundub, et sinu süsteem töötab virtuaalkeskkonnas (nt Linux-VServer), millel "
"puudub init-deemon. Ubuntu 10.04 LTS ei tööta sellises keskkonnas, vajalik "
"on eelnev virtuaalmasina seadistuse muutmine.\n"
"\n"
"Kas tahad sellest hoolimata jätkata?"

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Sandbox upgrade using aufs"
msgstr "Liivakastiuuendus kasutades aufs-i"

#: ../DistUpgrade/DistUpgradeMain.py:47
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Antud teelt otsitakse CD'd uuendatavate pakkidega"

#: ../DistUpgrade/DistUpgradeMain.py:53
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Liidese kasutamine. Hetkel saadaval: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:56
msgid "*DEPRECATED* this option will be ignored"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Soorita ainult osaline uuendamine (faili sources.list ei kirjutata üle)"

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid "Disable GNU screen support"
msgstr "Keela GNU ekraani tugi"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid "Set datadir"
msgstr "Määra andmekaust (datadir)"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:189
#: ../UpdateManager/DistUpgradeFetcherKDE.py:147
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Palun sisesta '%s' seadmesse '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "Allalaadimine on valmis"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "%li. faili %li-st allalaadimine kiirusel %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:295
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:301
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:361
#, python-format
msgid "About %s remaining"
msgstr "Umbes %s jäänud"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "%li. faili allalaadimine %li-st"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "Muudatuste rakendamine"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:211
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "sõltuvusprobleemid - jäetakse seadistamata"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:216
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "Paketi '%s' paigaldamine pole võimalik"

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Uuendamine jätkub, kuid pakett '%s' ei pruugi olla töökorras. Palun "
"raporteeri sellest veast."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:234
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Kas asendada muudetud seadistusfail\n"
"'%s'?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Kui asendad selle seadistusfaili uuema versiooniga, kaotad kõik sellesse "
"tehtud muudatused."

#: ../DistUpgrade/DistUpgradeViewGtk.py:250
#: ../DistUpgrade/DistUpgradeViewGtk3.py:253
#: ../DistUpgrade/DistUpgradeViewKDE.py:313
msgid "The 'diff' command was not found"
msgstr "Käsku 'diff' ei leitud"

#: ../DistUpgrade/DistUpgradeViewGtk.py:463
#: ../DistUpgrade/DistUpgradeViewGtk3.py:469
#: ../DistUpgrade/DistUpgradeViewText.py:90
msgid "A fatal error occurred"
msgstr "Esines saatuslik viga"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:470
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Palun raporteeri sellest kui veast (kui sa seda juba teinud pole) ja pane "
"raportiga kaasa /var/log/dist-upgrade/main.log ja /var/log/dist-"
"upgrade/apt.log failid. Uuendamine katkes.\n"
"Sinu esialgne sources.list fail salvestati nimega "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:481
#: ../DistUpgrade/DistUpgradeViewGtk3.py:487
msgid "Ctrl-c pressed"
msgstr "Vajutati Ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:488
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"See katkestab tegevuse ja võib jätta süsteemi katkisesse olekusse. Kas oled "
"kindel, et sa tahad seda teha?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:630
#: ../DistUpgrade/DistUpgradeViewGtk3.py:621
msgid "To prevent data loss close all open applications and documents."
msgstr "Andmekao vältimiseks sulge kõik avatud rakendused ja dokumendid."

#: ../DistUpgrade/DistUpgradeViewGtk.py:644
#: ../DistUpgrade/DistUpgradeViewGtk3.py:635
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Pole enam Canonicali poolt toetatud (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:636
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Paigaldatakse vanem (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:637
#, python-format
msgid "Remove (%s)"
msgstr "Eemaldatakse (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:638
#, python-format
msgid "No longer needed (%s)"
msgstr "Pole enam vaja (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:639
#, python-format
msgid "Install (%s)"
msgstr "Paigaldatakse (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:640
#, python-format
msgid "Upgrade (%s)"
msgstr "Uuendatakse (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:149
msgid "Media Change"
msgstr "Meedia muutmine"

#: ../DistUpgrade/DistUpgradeViewKDE.py:325
msgid "Show Difference >>>"
msgstr "Erinevuse näitamine >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:328
msgid "<<< Hide Difference"
msgstr "<<< Erinevuse peitmine"

#: ../DistUpgrade/DistUpgradeViewKDE.py:543
msgid "Error"
msgstr "Tõrge"

#: ../DistUpgrade/DistUpgradeViewKDE.py:555
msgid "&Cancel"
msgstr "&Loobu"

#: ../DistUpgrade/DistUpgradeViewKDE.py:557
#: ../DistUpgrade/DistUpgradeViewKDE.py:799
msgid "&Close"
msgstr "&Sulge"

#: ../DistUpgrade/DistUpgradeViewKDE.py:603
msgid "Show Terminal >>>"
msgstr "Terminali näitamine >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:606
msgid "<<< Hide Terminal"
msgstr "<<< Terminali peitmine"

#: ../DistUpgrade/DistUpgradeViewKDE.py:686
msgid "Information"
msgstr "Teave"

#: ../DistUpgrade/DistUpgradeViewKDE.py:737
#: ../DistUpgrade/DistUpgradeViewKDE.py:782
#: ../DistUpgrade/DistUpgradeViewKDE.py:785 ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Details"
msgstr "Üksikasjad"

#: ../DistUpgrade/DistUpgradeViewKDE.py:763
#, python-format
msgid "No longer supported %s"
msgstr "Pole enam toetatud %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:765
#, python-format
msgid "Remove %s"
msgstr "%s eemaldamine"

#: ../DistUpgrade/DistUpgradeViewKDE.py:767
#: ../DistUpgrade/DistUpgradeViewText.py:179
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Eemaldada (automaatselt paigaldatud) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:769
#, python-format
msgid "Install %s"
msgstr "%s paigaldus"

#: ../DistUpgrade/DistUpgradeViewKDE.py:771
#, python-format
msgid "Upgrade %s"
msgstr "%s uuendamine"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
#: ../DistUpgrade/DistUpgradeViewText.py:227
msgid "Restart required"
msgstr "Vajalik taaskäivitus"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr ""
"<b><big>Uuendamise lõpuleviimiseks palun taaskäivita süsteem</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:798 ../DistUpgrade/DistUpgrade.ui.h:25
#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid "_Restart Now"
msgstr "_Taaskäivita kohe"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:816
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Kas katkestada pooleliolev uuendus?</big></b>\n"
"\n"
"Uuenduse katkestamisega võib süsteem jääda mittekasutatavaks. Tungivalt "
"soovitatav on uuendamist jätkata."

#: ../DistUpgrade/DistUpgradeViewKDE.py:820
msgid "Cancel Upgrade?"
msgstr "Kas katkestada uuendamine?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li päev"
msgstr[1] "%li päeva"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li tund"
msgstr[1] "%li tundi"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minut"
msgstr[1] "%li minutit"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekund"
msgstr[1] "%li sekundit"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:80
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:98
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:147
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"See allalaadimine võtab umbes %s 1Mbit DSL ühendusega ja umbes %s 56k "
"modemiga."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Allalaadimine võtab sinu ühendusega aega umbes %s. "

#: ../DistUpgrade/DistUpgradeView.py:253 ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Preparing to upgrade"
msgstr "Uuenduse ettevalmistamine"

#: ../DistUpgrade/DistUpgradeView.py:254
msgid "Getting new software channels"
msgstr "Uute tarkvarakanalite hankimine"

#: ../DistUpgrade/DistUpgradeView.py:255 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Getting new packages"
msgstr "Uute pakettide hankimine"

#: ../DistUpgrade/DistUpgradeView.py:256 ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Installing the upgrades"
msgstr "Uuenduste paigaldamine"

#: ../DistUpgrade/DistUpgradeView.py:257 ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Cleaning up"
msgstr "Puhastamine"

#: ../DistUpgrade/DistUpgradeView.py:342
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d paigaldatud pakett pole enam Canonicali poolt toetatud. Seda "
"toetab siiski kogukond."
msgstr[1] ""
"%(amount)d paigaldatud paketti pole enam Canonicali poolt toetatud. Seda "
"toetab siiski kogukond."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:351
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakett eemaldatakse."
msgstr[1] "%d paketti eemaldatakse."

#: ../DistUpgrade/DistUpgradeView.py:356
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "Paigaldatakse %d uus pakett."
msgstr[1] "Paigaldatakse %d uut paketti."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakett uuendatakse."
msgstr[1] "%d paketti uuendatakse."

#: ../DistUpgrade/DistUpgradeView.py:367
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Alla tuleb laadida kokku %s. "

#: ../DistUpgrade/DistUpgradeView.py:371
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Uuenduse allalaadimine ja paigaldamine võib võtta mitu tundi. Kui "
"allalaadimine on lõppenud, pole protsessi enam võimalik katkestada."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:377 ../UpdateManager/UpdateManager.py:664
msgid "Your system is up-to-date"
msgstr "Süsteem on värske"

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Sinu süsteemile pole ühtegi uuendust saadaval. Uuendamisest loobutakse."

#: ../DistUpgrade/DistUpgradeView.py:391
msgid "Reboot required"
msgstr "Vajalik taaskäivitus"

#: ../DistUpgrade/DistUpgradeView.py:392
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Uuendamine on valmis ja vaja on süsteem taaskäivitada. Kas teha seda kohe?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid "Could not run the upgrade tool"
msgstr "Uuendamistööriista käivitamine polnud võimalik"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool signature"
msgstr "Uuendustööriista signatuur"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:214
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:214
msgid "Upgrade tool"
msgstr "Uuendustööriist"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Failed to fetch"
msgstr "Allalaadimine ebaõnnestus"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "Uuenduse allalaadimine ebaõnnestus. See võib olla võrguprobleem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:253
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:253
msgid "Failed to extract"
msgstr "Lahtipakkimine ebaõnnestus"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Uuenduse lahtipakkimine ebaõnnestus. Võrgu või serveriga võib probleeme "
"olla. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:259
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:259
msgid "Verification failed"
msgstr "Õigsuse kontroll nurjus"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:260
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:260
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Uuenduse ehtsuse kontroll ebaõnnestus. See võib olla võrgu- või "
"serveriprobleem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:265
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:265
msgid "Authentication failed"
msgstr "Autentimine ebaõnnestus"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:266
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:266
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Uuenduse autentimine ebaõnnestus. See võib olla võrgu- või serveriprobleem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid "Can not run the upgrade"
msgstr "Uuendamist pole võimalik käivitada"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:281
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:281
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:287
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:287
#, python-format
msgid "The error message is '%s'."
msgstr "Veateade on '%s'."

#: ../DistUpgrade/DistUpgradeViewText.py:91
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Palun raporteeri sellest kui veast ja pane raportiga kaasa /var/log/dist-"
"upgrade/main.log ja /var/log/dist-upgrade/apt.log failid. Uuendamine "
"katkes.\n"
"Sinu esialgne sources.list fail salvestati nimega "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:115
msgid "Aborting"
msgstr "Katkestamine"

#: ../DistUpgrade/DistUpgradeViewText.py:120
msgid "Demoted:\n"
msgstr "Degradeeritud:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:127
msgid "To continue please press [ENTER]"
msgstr "Jätkamiseks vajuta [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
#: ../DistUpgrade/DistUpgradeViewText.py:200
msgid "Continue [yN] "
msgstr "Kas jätkata? [y/N] "

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
msgid "Details [d]"
msgstr "Üksikasjad [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:159
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "y"
msgstr "y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:170
#, python-format
msgid "No longer supported: %s\n"
msgstr "Pole enam toetatud: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:175
#, python-format
msgid "Remove: %s\n"
msgstr "Eemaldatakse: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "Install: %s\n"
msgstr "Paigaldatakse: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Upgrade: %s\n"
msgstr "Uuendatakse: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:207
msgid "Continue [Yn] "
msgstr "Kas jätkata? [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:228
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Uuenduse lõpetamiseks on vajalik taaskäivitus.\n"
"Kui sa valid 'y', siis süsteem taaskäivitub."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Kas katkestada käimasolev uuendamine?</big></b>\n"
"\n"
"Kui uuendamise katkestad, võib süsteem olla kasutuskõlbmatu. On tungivalt "
"soovitatav uuendamist jätkata."

#: ../DistUpgrade/DistUpgrade.ui.h:5
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Uuenduse lõpetamiseks taaskäivita arvuti</big></b>\n"
"\n"
"Enne jätkamist palun salvesta pooleliolev töö."

#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Kas alustada uuendamist?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "<b><big>Upgrading Ubuntu to version 11.10</big></b>"
msgstr ""

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Difference between the files"
msgstr "Erinevus failide vahel"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Distribution Upgrade"
msgstr "Distributsiooniuuendus"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Restarting the computer"
msgstr "Arvuti taaskäivitamine"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Setting new software channels"
msgstr "Uute tarkvarakanalite seadistamine"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "Terminal"
msgstr "Terminal"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Cancel Upgrade"
msgstr "_Katkesta uuendamine"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Continue"
msgstr "_Jätka"

#: ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Replace"
msgstr "_Asenda"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Report Bug"
msgstr "_Teata veast"

#: ../DistUpgrade/DistUpgrade.ui.h:26
msgid "_Resume Upgrade"
msgstr "_Jätka uuendamist"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "_Start Upgrade"
msgstr "_Alusta uuendamist"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:55
msgid "Please wait, this can take some time."
msgstr "Palun oota, läheb natuke aega."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:57
msgid "Update is complete"
msgstr "Uuendamine on valmis"

#: ../UpdateManager/DistUpgradeFetcher.py:108
#: ../UpdateManager/DistUpgradeFetcherKDE.py:102
msgid "Could not find the release notes"
msgstr "Väljalaskemärkmeid ei leitud"

#: ../UpdateManager/DistUpgradeFetcher.py:109
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "The server may be overloaded. "
msgstr "Server võib olla ülekoormatud. "

#: ../UpdateManager/DistUpgradeFetcher.py:119
#: ../UpdateManager/DistUpgradeFetcherKDE.py:107
msgid "Could not download the release notes"
msgstr "Väljalaskemärkmete allalaadimine polnud võimalik"

#: ../UpdateManager/DistUpgradeFetcher.py:120
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Please check your internet connection."
msgstr "Palun kontrolli oma võrguühendust."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:61
#: ../UpdateManager/DistUpgradeFetcherKDE.py:84
msgid "Upgrade"
msgstr "Uuendamine"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:88
#: ../data/gtkbuilder/UpdateManager.ui.h:13
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Release Notes"
msgstr "Väljalaskemärkmed"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:127
#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
msgid "Downloading additional package files..."
msgstr "Täiendavate paketifailide allalaadimine..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "%s. fail %s-st kiirusel %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
#, python-format
msgid "File %s of %s"
msgstr "%s. fail %s-st"

#: ../UpdateManager/ChangelogViewer.py:73
msgid "Open Link in Browser"
msgstr "Ava link veebilehitsejas"

#: ../UpdateManager/ChangelogViewer.py:76
msgid "Copy Link to Clipboard"
msgstr "Kopeeri link"

#: ../UpdateManager/GtkProgress.py:159
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "%(current)li. faili allalaadimine %(total)li-st kiirusel %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:164
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "%(current)li. faili allalaadimine %(total)li-st"

#: ../UpdateManager/UpdateManager.py:105 ../do-release-upgrade:90
msgid "Your Ubuntu release is not supported anymore."
msgstr "Sinu Ubuntu väljalaset ei toetata enam."

#: ../UpdateManager/UpdateManager.py:106
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu Linux."
msgstr ""
"Sa ei saa edaspidi turvaparandusi ega kriitilisi uuendusi. Palun uuenda "
"mõnele Ubuntu Linuxi järgmisele versioonile."

#: ../UpdateManager/UpdateManager.py:114
msgid "Upgrade information"
msgstr "Uuenduse andmed"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:389
#, python-format
msgid "Version %s: \n"
msgstr "Versioon %s: \n"

#: ../UpdateManager/UpdateManager.py:445
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""
"Võrguühendust ei leitud, muudatuste logi allalaadimine pole võimalik."

#: ../UpdateManager/UpdateManager.py:453
msgid "Downloading list of changes..."
msgstr "Muudatuste nimekirja allalaadimine..."

#: ../UpdateManager/UpdateManager.py:497
msgid "_Deselect All"
msgstr "_Eemalda kõik"

#: ../UpdateManager/UpdateManager.py:503
msgid "Select _All"
msgstr "_Vali kõik"

#: ../UpdateManager/UpdateManager.py:552
#, python-format
msgid "%(count)s update has been selected."
msgid_plural "%(count)s updates have been selected."
msgstr[0] "Valitud on %(count)s uuendus."
msgstr[1] "Valitud on %(count)s uuendust."

#: ../UpdateManager/UpdateManager.py:556
#, python-format
msgid "%s will be downloaded."
msgstr "%s laaditakse alla."

#: ../UpdateManager/UpdateManager.py:568
msgid "The update has already been downloaded, but not installed"
msgid_plural "The updates have already been downloaded, but not installed"
msgstr[0] "See uuendus on juba alla laaditud, kuid pole paigaldatud"
msgstr[1] "Need uuendused on juba alla laaditud, kuid pole paigaldatud"

#: ../UpdateManager/UpdateManager.py:573
msgid "There are no updates to install"
msgstr "Pole ühtegi uuendust, mida paigaldada"

#. TRANSLATORS: this allows to switch the order of the count of
#. updates and the download size string (if needed)
#: ../UpdateManager/UpdateManager.py:579
#, python-format
msgid "%(count_str)s %(download_str)s"
msgstr "%(count_str)s %(download_str)s"

#: ../UpdateManager/UpdateManager.py:586
msgid "Unknown download size."
msgstr "Allalaadimise suurus pole teada."

#: ../UpdateManager/UpdateManager.py:612
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""

#: ../UpdateManager/UpdateManager.py:618
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Paketiandmeid uuendati viimati %(days_ago)s päeva tagasi.\n"
"Vajuta \"Kontrolli\" nuppu, et küsida tarkvarauuendusi."

#: ../UpdateManager/UpdateManager.py:623
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural ""
"The package information was last updated %(days_ago)s days ago."
msgstr[0] "Paketiandmeid uuendati viimati %(days_ago)s päev tagasi."
msgstr[1] "Paketiandmeid uuendati viimati %(days_ago)s päeva tagasi."

#: ../UpdateManager/UpdateManager.py:627
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] "Paketiandmeid uuendati viimati %(hours_ago)s tund tagasi."
msgstr[1] "Paketiandmeid uuendati viimati %(hours_ago)s tundi tagasi."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:632 ../UpdateManager/UpdateManager.py:634
#: ../UpdateManager/UpdateManager.py:636
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr "Paketiinfot uuendati viimati %s minuti eest."

#: ../UpdateManager/UpdateManager.py:638
msgid "The package information was just updated."
msgstr "Paketiinfot uuendati hetk tagasi."

#: ../UpdateManager/UpdateManager.py:658
#: ../data/gtkbuilder/UpdateManager.ui.h:22
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Tarkvarauuendused parandavad vigu, kõrvaldavad turvaauke ja lisavad uusi "
"funktsioone."

#: ../UpdateManager/UpdateManager.py:677
msgid "Software updates may be available for your computer."
msgstr "Sinu arvutile võivad olla kättesaadavad tarkvarauuendused."

#: ../UpdateManager/UpdateManager.py:685
msgid "Welcome to Ubuntu"
msgstr "Tere tulemast Ubuntusse"

#: ../UpdateManager/UpdateManager.py:687
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from Applications later."
msgstr ""

#: ../UpdateManager/UpdateManager.py:689
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from the Administration Menu later."
msgstr ""

#: ../UpdateManager/UpdateManager.py:692
msgid "Software updates are available for this computer"
msgstr "Selle arvuti tarkvarale on saadaval uuendusi"

#: ../UpdateManager/UpdateManager.py:694
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from "
"Applications later."
msgstr ""
"Kui sa ei soovi neid praegu paigaldada, käivita rakendus \"Uuendamishaldur\" "
"hiljem."

#: ../UpdateManager/UpdateManager.py:696
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Kui sa ei taha neid praegu paigaldada, võid seda teha hiljem valides "
"administreerimise menüüst \"Uuendamishalduri\"."

#: ../UpdateManager/UpdateManager.py:748
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Uuendamise jaoks on kokku vaja %s vaba ruumi kettal '%s' . Vabasta vähemalt "
"%s vaba ruumi seadmel '%s'. Tühjenda prügikast ja eemalda eelmiste "
"paigalduste ajutised pakid käsuga 'sudo apt-get clean'."

#: ../UpdateManager/UpdateManager.py:773
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Uuenduste paigaldamise lõpetamiseks tuleb arvuti taaskäivitada. Palun "
"salvesta enne jätkamist pooleliolevad tööd."

#: ../UpdateManager/UpdateManager.py:837
msgid "Reading package information"
msgstr "Paketiloendi lugemine"

#: ../UpdateManager/UpdateManager.py:852
msgid "Connecting..."
msgstr "Ühendumine..."

#: ../UpdateManager/UpdateManager.py:869
msgid "You may not be able to check for updates or download new updates."
msgstr "Võib-olla sa ei saa uuendusi kontrollida või alla laadida."

#: ../UpdateManager/UpdateManager.py:994
msgid "Could not initialize the package information"
msgstr "Paketiloendi lähtestamine polnud võimalik"

#: ../UpdateManager/UpdateManager.py:995
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Paketiinfo lähtestamisel esines lahendamatu viga.\n"
"\n"
"Palun raporteeri sellest kui paki 'update-manager' veast ning lisa raportile "
"järgnev teade:\n"

#: ../UpdateManager/UpdateManager.py:1024
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Uuenduse rehkendamisel esines lahendamatu viga.\n"
"\n"
"Palun raporteeri sellest kui paki 'update-manager' veast ning lisa raportile "
"järgnev teade:"

#: ../UpdateManager/UpdateManager.py:1049
msgid " (New install)"
msgstr " (uus paigaldus)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1056
#, python-format
msgid "(Size: %s)"
msgstr "(Suurus: %s)"

#: ../UpdateManager/UpdateManager.py:1058
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Versioonilt %(old_version)s versioonile %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1062
#, python-format
msgid "Version %s"
msgstr "Versioon %s"

#: ../UpdateManager/UpdateManager.py:1093 ../do-release-upgrade:102
msgid "Release upgrade not possible right now"
msgstr "Väljalaskele uuendamine ei ole praegu võimalik"

#: ../UpdateManager/UpdateManager.py:1094 ../do-release-upgrade:103
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Väljalaskele uuendamine ei ole praegu võimalik, proovi hiljem uuesti. Server "
"teatas: '%s'"

#: ../UpdateManager/UpdateManager.py:1096 ../check-new-release-gtk:116
msgid "Downloading the release upgrade tool"
msgstr "Väljalaske uuendamise tööriista allalaadimine"

#: ../UpdateManager/UpdateManager.py:1103
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Saadaval on uus Ubuntu väljalase '%s'</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1142
msgid "Software index is broken"
msgstr "Tarkvaraindeks on katki"

#: ../UpdateManager/UpdateManager.py:1143
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Tarkvara paigaldamine või eemaldamine on võimatu. Probleemi lahendamiseks "
"kasuta palun paketihaldur Synaptic'ut või terminalis käsku \"sudo apt-get "
"install -f\"."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Otsi uuendusi"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Paigalda kõik saadaval olevad uuendused"

#: ../UpdateManagerText/UpdateManagerText.py:31
msgid "Cancel"
msgstr "Katkesta"

#: ../UpdateManagerText/UpdateManagerText.py:32
msgid "Install"
msgstr "Paigalda"

#: ../UpdateManagerText/UpdateManagerText.py:50
msgid "Building Updates List"
msgstr "Uuenduste loendi koostamine"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Tavalist uuendust pole võimalik rehkendada, käivita \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Seda võib põhjustada:\n"
" * Eelmine pooleli jäänud uuendus\n"
" * Tõrked paigaldatud tarkvaraga\n"
" * Mitteametlikud pakid, mida ei jaga Ubuntu\n"
" * Ubuntu eelväljalaske tavapärased muutused"

#: ../UpdateManagerText/UpdateManagerText.py:122
msgid "Downloading changelog"
msgstr "Muudatuste logi allalaadimine"

#: ../UpdateManager/Core/MyCache.py:128
#, python-format
msgid "Other updates (%s)"
msgstr "Teised uuendused (%s)"

#: ../UpdateManager/Core/MyCache.py:283
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""
"Versioonide muutumised:\n"
"Paigaldatud versioon: %s\n"
"Saadaval olev versioon: %s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:297
msgid "This update does not come from a source that supports changelogs."
msgstr "See uuendus ei pärine muudatuste logisid toetavast allikast."

#: ../UpdateManager/Core/MyCache.py:303 ../UpdateManager/Core/MyCache.py:324
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Muudatuste nimekirja allalaadimine nurjus. \n"
"Palun kontrolli oma võrguühendust!"

#: ../UpdateManager/Core/MyCache.py:313
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Muudatuste logi ei sisalda mingeid asjakohaseid muudatusi.\n"
"\n"
"Senikaua, kuni muudatused logisse ilmuvad, oota natukene \n"
"või vaata http://launchpad.net/ubuntu/+source/%s/%s/+changelog"

#: ../UpdateManager/Core/MyCache.py:318
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Muudatuste logi pole veel saadaval.\n"
"\n"
"Senikaua, kuni muudatused logisse ilmuvad, oota natukene \n"
"või vaata http://launchpad.net/ubuntu/+source/%s/%s/+changelog"

#: ../UpdateManager/Core/UpdateList.py:49
msgid "Failed to detect distribution"
msgstr "Distributsiooni tuvastamine nurjus"

#: ../UpdateManager/Core/UpdateList.py:50
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Sinu kasutatava süsteemi kindlakstegemisel esines viga '%s'."

#: ../UpdateManager/Core/UpdateList.py:61
msgid "Important security updates"
msgstr "Tähtsad turvauuendused"

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Recommended updates"
msgstr "Soovitatavad uuendused"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Proposed updates"
msgstr "Pakutud uuendused"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Backports"
msgstr "Backport'id"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Distribution updates"
msgstr "Distributsiooniuuendused"

#: ../UpdateManager/Core/UpdateList.py:70
msgid "Other updates"
msgstr "Muud uuendused"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Sul tuleb uuendusi käsitsi kontrollida</big></b>\n"
"\n"
"Sinu süsteem ei kontrolli ise uuenduste saadavust. Seda saab muuta "
"<i>Tarkvaraallikatest</i>, mis asub <i>Uuenduste</i> paneelil."

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Mitte kõiki uuendusi ei saa paigaldada</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Töötab akutoitel</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Uuendamishalduri käivitamine</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Muudatused"

#: ../data/gtkbuilder/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "_Kontrolli"

#: ../data/gtkbuilder/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "_Jätka"

#: ../data/gtkbuilder/UpdateManager.ui.h:10
msgid "Description"
msgstr "Kirjeldus"

#: ../data/gtkbuilder/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Uuenduse kirjeldus"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "It’s safer to connect the computer to AC power before updating."
msgstr "On ohutum ühendada arvuti enne uuendamist vooluvõrku."

#: ../data/gtkbuilder/UpdateManager.ui.h:14
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Võimalikult paljude uuenduste paigaldamiseks võid teha osalise uuenduse.\n"
"\n"
"Seda võib põhjustada:\n"
" * Eelnev pooleli jäänud uuendus\n"
" * Probleemid paigaldatud tarkvaraga\n"
" * Mitteametlikud tarkvarapaketid, mida ei paku Ubuntu\n"
" * Tavapärased muudatused Ubuntu eelväljalaskes"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Show progress of individual files"
msgstr "Failide edenemise näitamine eraldi"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
msgid "Starting Update Manager"
msgstr "Uuendushalduri käivitamine"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
msgid "U_pgrade"
msgstr "_Uuenda"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
#: ../data/update-manager.desktop.in.h:3
msgid "Update Manager"
msgstr "Uuendamishaldur"

#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "You are connected via a wireless modem."
msgstr "Sa oled ühendatud juhtmeta modemi kaudu."

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Oled ühendatud läbi roaming-võrgu ja sulle võidakse esitada suur arve "
"uuenduse käigu allalaaditavate andmete suure mahu järgi."

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "Sinu arvuti töötab akutoitel. Kas oled kindel, et tahad jätkata?"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "_Hide this information in the future"
msgstr "_Peida tulevikus see info"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
msgid "_Install Updates"
msgstr "_Paigalda uuendused"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid "_Partial Upgrade"
msgstr "Uuenda _osaliselt"

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "_Settings..."
msgstr "_Seadistused"

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Upgrade"
msgstr "_Uuenda"

#: ../data/gtkbuilder/UpdateManager.ui.h:35
msgid "updates"
msgstr "uuendused"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Ubuntu uus versioon on saadaval. Kas tahad uuendada?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Ask Me Later"
msgstr "Küsi hiljem"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Ära uuenda"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "Yes, Upgrade Now"
msgstr "Jah, uuenda kohe"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr ""
"Sa võid uuendada ka hiljem avades uuenduste halduri ja klõpsates nupule "
"\"Uuenda\"."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Sa keeldusid uuele Ubuntu versioonile uuendamisest"

#: ../data/update-manager.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Näita ja paigalda saadaolevad uuendused"

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Tarkvara uuendused"

#: ../update-manager:54 ../do-release-upgrade:41
msgid "Show version and exit"
msgstr "Näitab versiooni ja väljub"

#: ../update-manager:57
msgid "Directory that contains the data files"
msgstr "Kataloog, mis sisaldab andmefaile"

#: ../update-manager:60
msgid "Check if a new Ubuntu release is available"
msgstr "Vaata, kas uus Ubuntu väljalase on saadaval"

#: ../update-manager:63 ../do-release-upgrade:44 ../check-new-release-gtk:176
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Kontrollib, kas viimasele arendusväljalaskele uuendamine on võimalik"

#: ../update-manager:67
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Uuendab väljalaskeuuendaja pakutud viimasele versioonile"

#: ../update-manager:74
msgid "Do not focus on map when starting"
msgstr "Käivitumisel ei ole fookus kaardil"

#: ../update-manager:77
msgid "Try to run a dist-upgrade"
msgstr "Üritatab käivitada dist-upgrade"

#: ../update-manager:80
msgid "Do not check for updates when starting"
msgstr "Käivitamisel ei kontrollita uuendusi"

#: ../update-manager:84 ../do-release-upgrade:60
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Testuuendus aufs-liivakastis"

#: ../update-manager:104
msgid "Running partial upgrade"
msgstr "Osalise uuendamise käivitamine"

#: ../do-release-upgrade:48 ../check-new-release-gtk:180
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Proovib uuendada viimasele versioonile kasutades $distro-proposed uuendajat"

#: ../do-release-upgrade:52
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Käivitub erilises uuendusrežiimis.\n"
"Praegu on toetatud 'desktop' tavalistele töölauaarvutitele ja 'server' "
"serversisüsteemide jaoks."

#: ../do-release-upgrade:58
msgid "Run the specified frontend"
msgstr "Käivita määratud liides"

#: ../do-release-upgrade:63
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Kontrollitakse ainult, kas uus distributsiooniväljalase on saadaval ja "
"teatatakse sellest väljumiskoodi kaudu"

#: ../do-release-upgrade:77
msgid "Checking for a new ubuntu release"
msgstr "Uue Ubuntu väljalaske kontrollimine"

#: ../do-release-upgrade:91
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Versiooniuuenduse info saamiseks külastage:\n"
"%(url)s\n"

#: ../do-release-upgrade:97
msgid "No new release found"
msgstr "Uut väljalaset ei leitud"

#: ../do-release-upgrade:109
#, c-format
msgid "New release '%s' available."
msgstr "Uus väljalase '%s' on saadaval."

#: ../do-release-upgrade:110
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Sellele uuendamiseks käivita 'do-release-upgrade'."

#: ../check-new-release-gtk:91
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Ubuntu %(version)s uuendus saadaval"

#: ../check-new-release-gtk:132
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Sa keeldusid Ubuntu uuendamisest versioonile %s"

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Teostamata meetod: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Fail kettal"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb pakett"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Puuduva paki paigaldamine."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Tuleks paigaldada pakett %s."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "Olekufailis on %i aegunud kirjet"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "Dpkg olekus on aegunud kirjeid"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "Dpkg olekukirjed on aegunud"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Kui uuendamise ajal on kdelibs4-dev paigaldatud, tuleb paigaldada ka "
"kdelibs5-dev. Täpsemalt aadressil bugs.launchpad.net, viga nr 279621."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s tuleb märkida käsitsipaigalduseks."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Eemalda lilo, kuna ka grub on paigaldatud. (Vaata vearaportit #314004)"

#~ msgid "0 KB"
#~ msgstr "0 KB"

#~ msgid "The window size"
#~ msgstr "Akna suurus"

#~ msgid ""
#~ "Stores the state of the expander that contains the list of changes and the "
#~ "description"
#~ msgstr "Salvestab sätteid, mis sisaldavad muudatusi ja kirjeldusi"

#~ msgid "Stores the size of the update-manager dialog"
#~ msgstr "update-manager'i dialoogiakna suurus salvestatakse"

#~ msgid "Show details of an update"
#~ msgstr "Uuenduste üksikasjade näitamine"

#~ msgid "Remind to reload the channel list"
#~ msgstr "Tuletatakse meelde, et kanalite nimekiri on vaja uuesti laadida"

#~ msgid ""
#~ "If automatic checking for updates is disabled, you have to reload the "
#~ "channel list manually. This option allows to hide the reminder shown in this "
#~ "case."
#~ msgstr ""
#~ "Kui uuenduste automaatne kontrollimine on väljalülitatud, pead kanalite "
#~ "nimekirja ise uuesti laadima. See valik võimaldab peita sel juhul näidatava "
#~ "meeldetuletuse."

#~ msgid "_Check All"
#~ msgstr "_Vali kõik"

#~ msgid "_Uncheck All"
#~ msgstr "Ära vali _midagi"

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug."
#~ msgstr ""
#~ "Varamuandmete uuendamisel loodi vigane fail. Palun teata sellest kui veast."

#~ msgid ""
#~ "It was impossible to install a required package. Please report this as a "
#~ "bug. "
#~ msgstr ""
#~ "Vajaliku paketi paigaldamine polnud võimalik. Palun teata sellest kui veast. "

#, python-format
#~ msgid "%.0f KB"
#~ msgstr "%.0f KB"

#~ msgid "1 KB"
#~ msgstr "1 KB"

#~ msgid ""
#~ "If none of this applies, then please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Kui ükski neist ei vasta tõele, raporteeri veast 'update-manager' paki kohta "
#~ "ning pane vearaportisse kõik failid kaustast /var/log/dist-upgrade/."

#~ msgid "The package information was last updated less than one hour ago."
#~ msgstr "Paketiloendit uuendati vähem kui tund aega tagasi."

#~ msgid "Show version in update list"
#~ msgstr "Uuendamise loendis näidatakse versiooninumbrit"

#, python-format
#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bug report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Palun raporteeri sellest 'update-manager' paki veast ja pane raportiga kaasa "
#~ "kõik failid /var/log/dist-upgrade/ kaustast.\n"
#~ "%s"

#, python-format
#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Pärast paketiinfo uuendamist ei leita enam hädavajalikku paketti '%s'.\n"
#~ "See viitab tõsisele veale. Palun raporteeri sellest kui 'update-manager' "
#~ "paketi veast ning pane vearaportiga kaasa kõik failid /var/log/dist-upgrade/ "
#~ "kasutast."

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this as a bug "
#~ "against the 'update-manager' package and include the files in /var/log/dist-"
#~ "upgrade/ in the bug report."
#~ msgstr ""
#~ "Süsteemi ettevalmistamine uuenduseks nurjus. Palun raporteeri sellest kui "
#~ "'update-manager' paketi veast ning pane vearaportiga kaasa kõik failid "
#~ "/var/log/dist-upgrade/ kasutast."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The upgrade "
#~ "will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug against the 'update-manager' package and include "
#~ "the files in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Süsteem ei suutnud täita uuenduse eeldusi. Uuendamine katkeb, taastatakse "
#~ "süsteemi esialgne olek.\n"
#~ "\n"
#~ "Palun raporteeri sellest kui 'update-manager' paketi veast ning pane "
#~ "vearaportiga kaasa kõik failid /var/log/dist-upgrade/ kasutast."

#~ msgid ""
#~ "This is most likely a bug in the upgrade tool. Please report it as a bug"
#~ msgstr ""
#~ "Tõenäoliselt on tegemist veaga uuendustööriistas. Palun raporteeri veast."

#~ msgid ""
#~ "This usually is caused by a system were /tmp is mounted noexec. Please "
#~ "remount without noexec and run the upgrade again."
#~ msgstr ""
#~ "Selle põhjus on tavaliselt süsteem, kus /tmp on haagitud noexec võtmega. "
#~ "Haagi see ilma noexec võtmeta ja käivita uuesti uuendus."

#~ msgid ""
#~ "It is unknown when the package information was updated last. Please try "
#~ "clicking on the 'Check' button to update the information."
#~ msgstr ""
#~ "See on teadmata, kuna paketiteavet uuendati hiljem. Teabe uuendamiseks "
#~ "klõpsa nupul 'Kontrolli'."

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration Menu later."
#~ msgstr ""
#~ "Need tarkvarauuendused on välja antud pärast Ubuntu väljalaset. Kui sa ei "
#~ "taha neid praegu paigaldada, saad seda hiljem teha Administeerimise menüüst "
#~ "\"Uuendamishaldur\" valides."

#~ msgid "Previous selected"
#~ msgstr "Eelmine valitud"

#~ msgid "*deprecated* Check for new distribution releases"
#~ msgstr "*aegunud* Kontrolli uusi distributsiooni väljalaskeid"

#~ msgid "If this key is set a first run welcome message will be presented."
#~ msgstr "Kui see võti on märgitud, näidatakse esmakäivituse tervitusteadet."

#~ msgid "First run welcome message"
#~ msgstr "Esmakäivituse tervitusteade"

#~ msgid "Auto close the install window after successful install"
#~ msgstr "See aken sulgub ise pärast edukat paigaldust"

#~ msgid ""
#~ "If this key is set the install window will be automatically closed on "
#~ "successful installation."
#~ msgstr ""
#~ "Kui see võti on märgitud, sulgub paigaldusaken pärast edukat paigaldust ise."

#~ msgid ""
#~ "This key is deprecated in favor of the file /etc/update-manager/release-"
#~ "upgrades Check automatically if a new version of the current distribution is "
#~ "available and offer to upgrade (if possible)."
#~ msgstr ""
#~ "See võti on aegunud, seda asendab fail /etc/update-manager/release-upgrades. "
#~ "Kontrollib automaatselt, kas selle distributsiooni uus väljalase on saadaval "
#~ "ning pakub uuendamist (kui võimalik)."

#~ msgid ""
#~ "If this key is set the main update list window will show version information "
#~ "(from version to version)."
#~ msgstr ""
#~ "Kui see võti on märgitud, kuvatakse uuenduste peaaknas versiooniteavet "
#~ "(milliselt millisele versioonile)."

#~ msgid "*DEPRECATED* this option will be ignore"
#~ msgstr "*AEGUNUD* seda valikut ignoreeritakse"

#, python-format
#~ msgid ""
#~ "Changes for the versions:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"
#~ msgstr ""
#~ "Versioonide muudatused:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"

#~ msgid "Settings..."
#~ msgstr "Sätted..."

#~ msgid ""
#~ "This will permanently hide the new release prompt from check-new-release-"
#~ "gtk. Note that the small button in the main update-manager UI will still be "
#~ "there."
#~ msgstr ""
#~ "See peidab jäädavalt uuendamise küsimuse check-new-release-gtkst. Sellest "
#~ "hoolimata jääb väike nupp update-manageri peaaknas alles."

#~ msgid "make check-new-release-gtk ignore a given new release"
#~ msgstr "check-new-release-gtk eiraku seda väljalaset"

#~ msgid "This change is not coming from a source that supports changelogs."
#~ msgstr "See muudatus ei pärine muudatuste logisid pakkuvast allikast."

#~ msgid "Show package summary before package name in update list"
#~ msgstr "Paketi kokkuvõtet kuvatakse uuenduste loendis enne paketi nime"

#~ msgid ""
#~ "If this key is set, the main update list window will show the package "
#~ "summary before the package name."
#~ msgstr ""
#~ "Kui see võti on määratud, näitab uuenduste peanimekiri paketi kokkuvõtet "
#~ "enne paketi nime."

#~ msgid ""
#~ "The upgrade has aborted. Please check your Internet connection or "
#~ "installation media and try again. All files downloaded so far are kept."
#~ msgstr ""
#~ "Uuendamine katkes. Kontrolli internetiühendust või paigaldusmeediumi ning "
#~ "proovi uuesti. Kõik seni allalaetud failid säilitatakse."

#~ msgid "<b><big>Upgrading Ubuntu to version 11.04</big></b>"
#~ msgstr "<b><big>Uuendamine Ubuntu versioonile 11.04</big></b>"

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from Applications "
#~ "later."
#~ msgstr ""
#~ "Need tarkvarauuendused on välja antud pärast Ubuntu väljalaset. Kui sa ei "
#~ "soovi neid praegu paigaldada, käivita rakendus \"Uuendamishaldur\" hiljem."
